test/e2e/network_policy_enforcement.go

```
@@ -0,0 +1,233 @@
+		g.GinkgoWriter.Printf("deleting test namespace %s\n", clientNS)
+		_ = kubeClient.CoreV1().Namespaces().Delete(ctx, clientNS, metav1.DeleteOptions{})
+	})
+	expectConnectivity(ctx, kubeClient, clientNS, map[string]string{"test": "any-pod"}, serverIPs, 8080, true)
+
+	g.By("Verifying allowed labels can reach server within namespace")
+	expectConnectivity(ctx, kubeClient, ns, allowedClientLabels, serverIPs, 8080, true)
+
+	g.By("Verifying unmatched labels cannot reach server (egress blocked by default-deny)")
+	expectConnectivity(ctx, kubeClient, ns, map[string]string{"role": "denied"}, serverIPs, 8080, false)
+}
+
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofumpt)